### PR TITLE
Add PAD Tips and Scrum question

### PR DESCRIPTION
**Pull request by NutthanichN**
I create a question about PAD Tips and Scrum concepts, please review it. Thank you.

### CODE REVIEW

File: pad-tips-and-scrum/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+## PAD Tips and Scrum
+
+### Question
+
+Given a list of Tips from PAD in the table below:
+
+| Tip's number| Name from *Practices of an Agile Developer*  | Name from PAD refcard                       |
```
Comment (jbrucker) [inline pad-tips-and-scrum/README.md]: Tip's number -> Tip number

File: pad-tips-and-scrum/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+| Tip 17      | *Use Short Iterations, Release in Increments*| *Develop in increments*                     |
+| Tip 18      | *Fixed Prices Are Broken Promises*           | *Estimate based on real work*               |
+| Tip 23      | *Measure Real Progress*                      | *Measure how much work is left*             |
+| Tip 28      | *Code in Increment*                          | *Write code in short edit/build/test cycles*|
+| Tip 38      | *Schedule Regular Face Time*                 | *Use stand-up meetings*                     |
+| Tip 45      | *Keep Others Informed*                       | *Keep others informed*                      |
+
+Which Tip is suitable the most for each Scrum concept below? Answer only one Tip for each Scrum concept.
+
+1. _____ : Product owner makes sure that right user stories are in the product backlog.
```
Comment (jbrucker) [inline pad-tips-and-scrum/README.md]: The colons look out of place.  Suggest deleting the colons.

File: pad-tips-and-scrum/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+|-------------|----------------------------------------------|---------------------------------------------|
+| Tip 10      | *Let Customers Make Decisions*               | *Let your customers decide*                 |
+| Tip 17      | *Use Short Iterations, Release in Increments*| *Develop in increments*                     |
+| Tip 18      | *Fixed Prices Are Broken Promises*           | *Estimate based on real work*               |
+| Tip 23      | *Measure Real Progress*                      | *Measure how much work is left*             |
+| Tip 28      | *Code in Increment*                          | *Write code in short edit/build/test cycles*|
+| Tip 38      | *Schedule Regular Face Time*                 | *Use stand-up meetings*                     |
+| Tip 45      | *Keep Others Informed*                       | *Keep others informed*                      |
+
+Which Tip is suitable the most for each Scrum concept below? Answer only one Tip for each Scrum concept.
```
Comment (jbrucker) [inline pad-tips-and-scrum/README.md]: "suitable the most" -> "most suitable",
but I think "Which Tip best describes each Scrum activity"

File: pad-tips-and-scrum/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+1. _____ : Product owner makes sure that right user stories are in the product backlog.
+2. _____ : Team members answer three questions in Daily Scrum.
+3. _____ : Use a burndown chart to monitor the progress of sprint.
+4. _____ : Daily Scrum.
+5. _____ : Split user stories in release backlog to sprints.
```
Comment (jbrucker) [inline pad-tips-and-scrum/README.md]: "Split" implies subdividing.  Do you mean "Assign"?

File: pad-tips-and-scrum/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+| Tip 23      | *Measure Real Progress*                      | *Measure how much work is left*             |
+| Tip 28      | *Code in Increment*                          | *Write code in short edit/build/test cycles*|
+| Tip 38      | *Schedule Regular Face Time*                 | *Use stand-up meetings*                     |
+| Tip 45      | *Keep Others Informed*                       | *Keep others informed*                      |
+
+Which Tip is suitable the most for each Scrum concept below? Answer only one Tip for each Scrum concept.
+
+1. _____ : Product owner makes sure that right user stories are in the product backlog.
+2. _____ : Team members answer three questions in Daily Scrum.
+3. _____ : Use a burndown chart to monitor the progress of sprint.
```
Comment (jbrucker) [inline pad-tips-and-scrum/README.md]: "of sprint" -> "of the Sprint"

File: pad-tips-and-scrum/README.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+4. _____ : Daily Scrum.
+5. _____ : Split user stories in release backlog to sprints.
+
+### Answer
+
+1. Tip 10 : Product owner makes sure that right user stories are in the product backlog.
```
Comment (jbrucker) [inline pad-tips-and-scrum/README.md]: No space before colon.  I've never seen a case where this is valid English punctuation.

File: pad-tips-and-scrum/README.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+5. Tip 17 : Split user stories in release backlog to sprints.
+
+Notes
+- Tip 18 *Estimate based on real work* is about estimating accomplished works of the project to inform the customer.
+- Tip 28 *Write code in short edit/build/test cycles* is about a good coding practice.
+
+**References**
+- *Practices of an Agile Developer* by Venkat Subramaniam and Andy Hunt
+
+   - Tip 10 on page 45
```
Comment (jbrucker) [inline pad-tips-and-scrum/README.md]: I think giving page numbers is not necessary.
Let them go look it up!  In an e-book the page numbers
might not match the printed book, anyway. (ePub format
doesn't even have page numbers, I think.)

File: pad-tips-and-scrum/README.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+   - Tip 17 on page 69
+   - Tip 18 on page 73
+   - Tip 23 on page 93
+   - Tip 28 on page 113
+   - Tip 38 on page 148
+   - Tip 45 on page 168
+- [Intro to Scrum in Under 10 Minutes (video)](https://www.youtube.com/watch?v=XU0llRltyFM&feature=youtu.be)
+- [Introduction to Scrum](https://cpske.github.io/ISP/agile/Intro-Scrum-MountainGoat.pdf) by Mountain Goat Software
```
Comment (jbrucker) [inline pad-tips-and-scrum/README.md]: These are all Scrum links I gave the class.
Second link should be actual link to the document 
on MontainGoat's web site, not a link to someone's
repost of their document. (3rd party reference)

Can you find at least one other *good* scrum link
that applies to your question?

**Comment by NutthanichN** [comment]
> 
> 
> This is a good question, but please look at my comments
> and either fix them or explain why it should stay as is.

I edit my question based on the suggestions and add more Scrum reference. Please review it, thank you.

**Comment by jbrucker** [comment]
Looks good now.

**Pull request merged**